### PR TITLE
updated the demo app, removed deprecated SDKs

### PR DESCRIPTION
**Pull request by Z3nAjo**
## Changes
* Min version to 21
* Target to 33 (exception for the ua)
* Connect removed
* Chat v1 removed
* All dependencies regrouped in the main Gradle file to reduce maintenance effore

### Reviewers
* @zendesk/menerva 
* @michaelodonnell

### CODE REVIEW

File: answerbot_sample/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
     defaultConfig {
-        applicationId rootProject.ext.applicationId + ".answerbot"
+        applicationId rootProject.ext.applicationId + ".answerbot_sample"
```
Comment (ReguloSar) [inline answerbot_sample/build.gradle:7]: Just curious why we updated the application id for this.

File: support_override_styles/src/main/java/bfahy/support_override_styles/Global.kt [hunk 2]
```diff
@@ -10,9 +10,9 @@ import zendesk.support.Support
 
 class Global : Application() {
 
-    private val SUBDOMAIN_URL = ""
-    private val APPLICATION_ID = ""
-    private val OAUTH_CLIENT_ID = ""
+    private val SUBDOMAIN_URL = "https://z3nadvocate84.zendesk.com"
```
Comment (ReguloSar) [inline support_override_styles/src/main/java/bfahy/support_override_styles/Global.kt:13]: should we provide these credentials here?

File: messaging_sample/src/main/java/com/zendesk/messaging_sample/Global.java [hunk 3]
```diff
@@ -62,10 +63,9 @@ public void onCreate() {
         );
 
         // b). JWT (Must be initialized with your JWT identifier)
-        Zendesk.INSTANCE.setIdentity(new JwtIdentity("{JWT User Identifier}"));
-
+        //Zendesk.INSTANCE.setIdentity(new JwtIdentity("{JWT User Identifier}"));
```
Comment (ReguloSar) [inline messaging_sample/src/main/java/com/zendesk/messaging_sample/Global.java:66]: is this intended?

**Review by ReguloSar — APPROVED** [review]
Really nice work, Arnaud. You've even upgraded the gradle version as well.  Just left minor comments.

File: answerbot_sample/build.gradle (hunk 1 above)
Comment (Z3nAjo) [reply answerbot_sample/build.gradle:7]: just to have the same name as the package, and that way all app have the same applicationId format

File: support_override_styles/src/main/java/bfahy/support_override_styles/Global.kt (hunk 2 above)
Comment (Z3nAjo) [reply support_override_styles/src/main/java/bfahy/support_override_styles/Global.kt:13]: I forgot to remove, will remove that

File: messaging_sample/src/main/java/com/zendesk/messaging_sample/Global.java (hunk 3 above)
Comment (Z3nAjo) [reply messaging_sample/src/main/java/com/zendesk/messaging_sample/Global.java:66]: Yes, in the past some customer were confused why some app would crash as the identity becomes JWT if we leave this in, and most of them don't test with it setup. Just to error-proof the app more, I think it's clear enough to have the "basic" option there and in comment the more advanced one

**Comment by Z3nAjo** [comment]
@michaelodonnell the changes brought break the chat-providers sample, the message bubbles are not displayed and the image loading library has issues as well. Either we fix it but we'll have to spend more time on it or we can remove it

**Comment by Z3nAjo** [comment]
@michaelodonnell I also didn't update the urban airship app above API 29 (as the UA library needs to be updated by a lot of versions) - out of consistency concern we can spend time on it, but barely anyone uses UA (I think they number like half a dozen worldwide and no major account)

**Review by loyannec — APPROVED** [review]

**Comment by Z3nAjo** [comment]
@michaelodonnell as discussed, removed:
- Chat v1 samples
- Connect sample
- Chat v2 provider sample
Bumped everything to Android 13 except the Urban airship that is still API 29

**Pull request merged**